cdap-security/src/test/resources/cdap-security.xml

```diff
@@ -31,4 +31,9 @@
       <value>JKS</value>
       <description>Specifies the ssl keystore type</description>
     </property>
+    <property>
```

Comment:
It doesn't. This is only for testing. In deployment the user will have to create a cdap-security.xml and add the required properties. We don't want to provide a default password.